wgpu-bindings/Cargo.toml

```diff
@@ -10,4 +10,5 @@ authors = [
 default = []
 
 [dependencies]
-cbindgen = { git = "https://github.com/eqrion/cbindgen.git" }
+#cbindgen = "0.6.7"
+cbindgen = { git = "https://github.com/eqrion/cbindgen" }
```

Comment:
We could also use [my branch](https://github.com/eqrion/cbindgen/pull/248) for now if we're blocked by the header changes